### PR TITLE
IAC-753 - Doc review

### PR DESCRIPTION
**Pull request by supear**
Edited docs in repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Regardless of which method you choose, the machines in your deployment must meet
 - At least 6 machines for the application nodes in your cluster
 - 1 machine to serve as a jump server
 - 1 machine to serve as an NFS server
-- (Optional) At least 1 machine to host a PostgreSQL server (for the SAS Infrastructure Data Server component) if you plan to use an external database server with your cluster.
+- (Optional) At least 1 machine to host a PostgreSQL server (for the SAS Infrastructure Data Server component) if you plan to use an external database server with your cluster.
```
Comment (thpang) [inline README.md:54]: Lines have an extra space or line ending. Please remove.

File: README.md [hunk 2]
```diff
@@ -106,7 +106,8 @@ The following software is required in order to run the SAS Viya IaC tools here o
 
 #### Docker Requirements
 
-If you are using the predefined dockerfile in this project in order to run the script, you need only an instance of
+If you are using the predefined dockerfile in this project in order to run the script, you need only an instance of 
+
```
Comment (thpang) [inline README.md:110]: Line needs to be removed formatting corrected.

File: README.md [hunk 3]
```diff
@@ -129,7 +130,7 @@ cd viya4-iac-k8s
 
 #### vSphere/vCenter Machines
 
-Terraform scripts require variable definitions as input. Review the variables files and modify default values to meet your requirements.
+Terraform scripts require variable definitions as input. Review the variables files and modify default values to meet your requirements.
```
Comment (thpang) [inline README.md:133]: Lines have an extra space or line ending. Please remove.

File: README.md [hunk 4]
```diff
@@ -139,7 +140,7 @@ You have the option to specify variable definitions that are not included in `te
 
 #### SAS Viya IaC Configuration Files
 
-In order to use this repository, modify the [inventory file](./examples/bare-metal/sample-inventory) to provide information about the machine targets for the SAS Viya deployment.
+In order to use this repository, modify the [inventory file](./examples/bare-metal/sample-inventory) to provide information about the machine targets for the SAS Viya deployment.
```
Comment (thpang) [inline README.md:143]: Lines have an extra space or line ending. Please remove.

**Review by thpang — CHANGES_REQUESTED** [review]

**Comment by thpang** [comment]
Added a few folks to also review. ;)

**Review by thpang — APPROVED** [review]

**Review by jarpat — APPROVED** [review]

**Review by riragh — APPROVED** [review]

**Pull request merged**